blueos_repository/consolidate.py

```diff
@@ -146,24 +172,33 @@ async def run(self) -> None:
                         continue
                     raw_labels = await self.registry.fetch_labels(f"{repository.docker}:{tag}")
                     permissions = raw_labels.get("permissions", None)
-                    website = raw_labels.get("website", None)
+                    links = raw_labels.get("links", {})
+                    website = links.pop("website", raw_labels.get("website", None))
                     authors = raw_labels.get("authors", None)
-                    docs = raw_labels.get("docs", None)
+                    docs = links.pop("docs", links.pop("documentation", raw_labels.get("docs", None)))
                     readme = raw_labels.get("readme", None)
                     if readme is not None:
                         readme = readme.replace(r"{tag}", tag)
-                    company_raw = raw_labels.get("company", None)
-                    company = Company.from_json(json.loads(company_raw)) if company_raw is not None else None
-                    support = raw_labels.get("support", None)
+                    # support new "maintainer" and old "company" name for label
+                    maintainer_raw = raw_labels.get("maintainer", raw_labels.get("company", None))
```

Comment:
That'll get convoluted in the future if we continue changing it.

Reply:
Agreed, but field names aren’t expected to change much, there’s just been a more cohesive effort put into thinking through what the fields should actually be, compared to the original rough ones Willian started with.

Reply:
This has been pulled out for now anyway.